### DISCUSSION TITLE
tailwind css not working with react after v3.2.1 releases

### DISCUSSION THREAD

**Discussion by mickeymaruf** [tailwindlabs/tailwindcss#9647] [Help]
when i installed latest tailwind in my react app as usual by npm i tailwindcss with the other dependency, it didn't work. even i tried with different apps but same issue. then heard that some of my friends are facing the same thing. I tried version v3.1.8 working perfectly. Please fix the issue with the latest version i think 3.2.1.

**Comment by wongjn** [comment]
Could you provide more details about how it is not working? Providing a repository that exhibits the undesired behavior can help with troubleshooting too.

**Reply by mickeymaruf** [reply]
It's just not getting those css classes of tailwind, not even a single class. here is the repo i was working with:
https://github.com/mickeymaruf/travel-edge
**NOTE**: i was using tailwind cdn for temporarily, you can remove/comment it from public/index.html, just above the title and let me know why i'm causing the issue. TIA :)

**Reply by wongjn** [reply]
Thank you for providing that. I took the repository for a quick spin and it all seems to work fine removing the Tailwind CDN `<script>` and running `npm start` – was there any specific steps that need to be followed to exhibit the undesired behavior?

**Reply by mickeymaruf** [reply]
I don't know, but some of my friends are facing the same issue, and some of them are not. so I don't know what's the specific reason behind it. but as i've started using the tailwind v3.1.8 now it's fine, I guess the issue will be solved after the tailwind stable release of v3.2.1, hope so. again Thank's for your time man.

**Comment by thecrypticace** [comment]
Hey! Can you give our insiders build a try to see if this issue is fixed for you? `npm install tailwindcss@insiders`

**Reply by mickeymaruf** [reply]
yeah, it solves the problem.
